spec/lib/appsignal/hooks/excon_spec.rb

```diff
@@ -0,0 +1,62 @@
+describe Appsignal::Hooks::ExconHook do
+  before :context do
+    start_agent
+  end
+
+  context "with Excon" do
+    before(:context) do
+      class Excon
+        def self.defaults
+          @defaults ||= {}
+        end
+      end
+
+      Appsignal::Hooks::ExconHook.new.install
+    end
+    after(:context) { Object.send(:remove_const, :Excon) }
+
+    describe "#dependencies_present?" do
+      subject { described_class.new.dependencies_present? }
+
+      it { is_expected.to be_truthy }
+    end
+
+    it "should instrument a http request" do
+      Appsignal::Transaction.create("uuid", Appsignal::Transaction::HTTP_REQUEST, "test")
```

Comment:
I'll probably rewrite this to use the `transaction.to_h` format for the specs before merging.